doc/gov/chapters/chapter09.md

```diff
@@ -81,3 +81,322 @@ Exceptions will be clearly recorded in Reference Architectures' Appendices which
+
+<a name="9.5.2.2"></a>
+### 9.5.2.2 Purpose of this Document Section
+
+The purpose of this document is to define the goals/outcomes, expectations, and roles necessary to support the CNTT release trials. The document will define/discuss the following:
+
+*	Purpose of field trials
+*	Goals/desired outcomes of the field trials
+*	Success indicators 
+*	Intentions and expectations
+*	Action plan
+*	Resource requirements
+*	Metrics definition
+
+<a name="9.5.2.3"></a>
+### 9.5.2.3 Field Trial Purpose
+
+In the truest form, adoption of a standard is an indication of it’s success within an industry. Standards developed must be interactively tested in multiple environments or “trialed” to ensure they are practicable, functional, and operative. Without running trials to validate the CNTT approach, standards may not provide intended value across a sufficient spectrum of participating entities to be widely adopted. 
+
+<a name="9.5.2.4"></a>
+#### 9.5.2.4 Intents
+
+1)	This is a partnership approach to validate CNTT community is adopting a consistent approach
+2)	Validate RI-1 specifications and RC-1 test suite, not VNFs or NFVI’s in the trial
+
+<a name="9.5.2.5"></a>
+#### 9.5.2.5 Key Expectations and Assumptions
+
+1)	Expectation is through healthy feedback from suppliers, CNTT will exit the trial with either validation of RI-1 and RC-1 or a set of actions to close gaps.
+2)	Expectation: Post trial and gap closure, the community will define a badging process that is mutually beneficial to operators and suppliers. 
+3)	Assumption: Performance testing is not in field trial
+
+<a name="9.5.2.6"></a>
+### 9.5.2.6 Overview: Stages of Field Trial
+
+The following diagram the key components and flow of activities, actions, and deliverables to be undertaken during the trial. Details of each component are provided in this document.
+
+<p align="center"><img src="../figures/field_trials.png" alt="CNTT Field Trial Approach" title="Field Trial Approach" width="85%"/></p>
+<p align="center"><b>Figure 3:</b> Field Trial Approach</p>
+
+<a name="9.5.2.7"></a>
+#### Success Indicators
+
+1) Agreement secured on the use of trials results data, including:
+a.	Detail level of data required to validate
+b.	Acceptable data values indicating valid results
+c.	Detail level of data that will be published
+
+2) RI-1 Labs are successfully stood up in all target environments
+* Vendor (3rd party)
+*	Community (OPNFV)
+*	LaaS (e.g. UNH)
+
+3) Engaged NVFI vendors successfully configure their infrastructure per RI-1 and run the RC-1 test suite and are able to provide expert feedback
+
+4) Engaged VNF vendors are able to validate that they can instantiate and run rudimentary validation of VNF functionality on more than one conformant NVFI
```

Comment:
Can engaged VNF vendors create their own infrastructure and configure as per RI-1?

Can OpenStack distributors do the same?

Essentially, that the various category of vendors can create their own labs

Reply:
Resolved